### DISCUSSION TITLE
The default benchmark TPCH data size is 1gb, how do I set the benchmark TPCH data size to 10 gb or 50 gb?

### DISCUSSION THREAD

**Discussion by ymkzpx** [duckdb/duckdb#4777] [Q&A] [answered]
The default benchmark TPCH data size is 1gb, how do I set the benchmark TPCH data size to 10 gb or 50 gb?

Looking forward to your reply, Thank you.

**Comment by Alex-Monahan** [comment]
I apologize if I am incorrect (I do not know C++!), but I think you can edit the variable SF in this file!
https://github.com/duckdb/duckdb/blob/master/benchmark/tpch/startup.cpp

**Comment by lnkuiper** [comment, marked as answer]
From SQL you can do:
```sql
CALL dbgen(sf=10);
```
